### PR TITLE
Added AdomisTheOracle, VessTheOracle and YulukTheOracle

### PR DESCRIPTION
**Pull request by Zayberex**
+AdomisTheOracle
+VessTheOracle
+YulukTheOracle
+updated repo

### CODE REVIEW

File: game/cards/dm06/light_bringer.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		ctx.Match.ShowCards(
+			card.Player,
+			"Your shield:",
+			ids,
+		)
+
+		card.Tapped = true
+
+	}))
+}
```
Comment (sindreslungaard) [inline game/cards/dm06/light_bringer.go:48]: I think the card text means it should be possible to choose a card from the opponent's shieldzone too, unless there's some rulings i'm not aware of

File: game/cards/dm06/light_bringer.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	c.Civ = civ.Light
+	c.Family = []string{family.LightBringer}
+	c.ManaCost = 1
+	c.ManaRequirement = []string{civ.Light}
+
+	spellcast := false
+
+	c.Use(fx.Creature, func(card *match.Card, ctx *match.Context) {
+
+		ctx.Match.ApplyPersistentEffect(func(ctx2 *match.Context, exit func()) {
```
Comment (sindreslungaard) [inline game/cards/dm06/light_bringer.go:75]: I don't think we need to use a persistent effect here, should be enough to listen to SpellCast and EndOfTurn event to set spellcast to true/false. Also, a new persistent effect is being added for every event that is being fired in the match here which is probably not what was intended

**Review by sindreslungaard — CHANGES_REQUESTED** [review]

File: game/cards/dm06/light_bringer.go (hunk 1 above)
Comment (Zayberex) [reply game/cards/dm06/light_bringer.go:48]: Oh, it shoud actually, but with that i dont know if i can make it work

**Comment by sindreslungaard** [comment]
Added in #238

**Pull request closed without merging**